.gitlab-ci.yml

```diff
@@ -1051,10 +1051,13 @@ workflow:
       compare_to: main # TODO: use a variable, when this is supported https://gitlab.com/gitlab-org/gitlab/-/issues/369916
 
 .on_fakeintake_changes:
-  - <<: *fakeintake_changes
+  - changes:
+      <<: *fakeintake_paths
+      compare_to: main # TODO: use a variable, when this is supported https://gitlab.com/gitlab-org/gitlab/-/issues/369916
 
 .on_fakeintake_changes_on_main:
-  - <<: *fakeintake_changes
+  - changes:
+      <<: *fakeintake_paths
```

Comment:
`compare_to: main # ...` is missing here

Reply:
It's on purpose - this rule only runs on main, so if you add `compare_to: main`, then this is equivalent to the rule never triggering.

Reply:
Oh okay I see, thanks 👍